[2022-11-07T15:47:28.455+0000] {processor.py:156} INFO - Started process (PID=51) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:47:28.459+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:47:28.461+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:47:28.460+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:47:28.625+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:47:28.607+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/forex_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/forex_data_pipeline.py", line 18, in <module>
    owner_links={"felipehdez": "juanfelipehdezm@gmail.com"}, catchup=False) as dag:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 583, in __init__
    "Wrong link format was used for the owner. Use a valid link \n"
airflow.exceptions.AirflowException: Wrong link format was used for the owner. Use a valid link 
Bad formatted links are: {'felipehdez': 'juanfelipehdezm@gmail.com'}
[2022-11-07T15:47:28.630+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:47:28.661+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.209 seconds
[2022-11-07T15:47:58.766+0000] {processor.py:156} INFO - Started process (PID=70) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:47:58.772+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:47:58.774+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:47:58.773+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:47:58.902+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:47:58.878+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/forex_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/forex_data_pipeline.py", line 18, in <module>
    owner_links={"felipehdez": "juanfelipehdezm@gmail.com"}, catchup=False) as dag:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 583, in __init__
    "Wrong link format was used for the owner. Use a valid link \n"
airflow.exceptions.AirflowException: Wrong link format was used for the owner. Use a valid link 
Bad formatted links are: {'felipehdez': 'juanfelipehdezm@gmail.com'}
[2022-11-07T15:47:58.910+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:47:58.936+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.173 seconds
[2022-11-07T15:48:29.267+0000] {processor.py:156} INFO - Started process (PID=97) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:48:29.272+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:48:29.273+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:48:29.273+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:48:29.333+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:48:29.322+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/forex_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/forex_data_pipeline.py", line 18, in <module>
    owner_links={"felipehdez": "juanfelipehdezm@gmail.com"}, catchup=False) as dag:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 583, in __init__
    "Wrong link format was used for the owner. Use a valid link \n"
airflow.exceptions.AirflowException: Wrong link format was used for the owner. Use a valid link 
Bad formatted links are: {'felipehdez': 'juanfelipehdezm@gmail.com'}
[2022-11-07T15:48:29.337+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:48:29.360+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.095 seconds
[2022-11-07T15:48:59.612+0000] {processor.py:156} INFO - Started process (PID=125) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:48:59.615+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:48:59.617+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:48:59.617+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:48:59.706+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:48:59.686+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/forex_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/forex_data_pipeline.py", line 18, in <module>
    owner_links={"felipehdez": "juanfelipehdezm@gmail.com"}, catchup=False) as dag:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 583, in __init__
    "Wrong link format was used for the owner. Use a valid link \n"
airflow.exceptions.AirflowException: Wrong link format was used for the owner. Use a valid link 
Bad formatted links are: {'felipehdez': 'juanfelipehdezm@gmail.com'}
[2022-11-07T15:48:59.713+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:48:59.739+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.131 seconds
[2022-11-07T15:49:05.789+0000] {processor.py:156} INFO - Started process (PID=126) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:49:05.793+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:49:05.794+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:05.794+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:49:05.901+0000] {logging_mixin.py:117} INFO - hola
[2022-11-07T15:49:05.924+0000] {processor.py:768} INFO - DAG(s) dict_keys(['forex_data_pipeline']) retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:49:06.108+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:06.108+0000] {manager.py:507} INFO - Created Permission View: can edit on DAG:forex_data_pipeline
[2022-11-07T15:49:06.118+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:06.118+0000] {manager.py:507} INFO - Created Permission View: can read on DAG:forex_data_pipeline
[2022-11-07T15:49:06.130+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:06.130+0000] {manager.py:507} INFO - Created Permission View: can delete on DAG:forex_data_pipeline
[2022-11-07T15:49:06.131+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:06.131+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T15:49:06.152+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:06.152+0000] {dag.py:2594} INFO - Creating ORM DAG for forex_data_pipeline
[2022-11-07T15:49:06.167+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:06.167+0000] {dag.py:3328} INFO - Setting next_dagrun for forex_data_pipeline to 2022-11-07T00:00:00+00:00, run_after=2022-11-08T00:00:00+00:00
[2022-11-07T15:49:06.199+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.413 seconds
[2022-11-07T15:49:36.547+0000] {processor.py:156} INFO - Started process (PID=154) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:49:36.551+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:49:36.553+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:36.553+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:49:36.615+0000] {logging_mixin.py:117} INFO - hola
[2022-11-07T15:49:36.626+0000] {processor.py:768} INFO - DAG(s) dict_keys(['forex_data_pipeline']) retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:49:36.684+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:36.684+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T15:49:36.723+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:49:36.722+0000] {dag.py:3328} INFO - Setting next_dagrun for forex_data_pipeline to 2022-11-07T00:00:00+00:00, run_after=2022-11-08T00:00:00+00:00
[2022-11-07T15:49:36.758+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.214 seconds
[2022-11-07T15:50:06.824+0000] {processor.py:156} INFO - Started process (PID=173) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:50:06.829+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:50:06.831+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:50:06.831+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:50:06.888+0000] {logging_mixin.py:117} INFO - hola
[2022-11-07T15:50:06.898+0000] {processor.py:768} INFO - DAG(s) dict_keys(['forex_data_pipeline']) retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:50:06.993+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:50:06.993+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T15:50:07.019+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:50:07.019+0000] {dag.py:3328} INFO - Setting next_dagrun for forex_data_pipeline to 2022-11-07T00:00:00+00:00, run_after=2022-11-08T00:00:00+00:00
[2022-11-07T15:50:07.041+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.221 seconds
[2022-11-07T15:50:37.378+0000] {processor.py:156} INFO - Started process (PID=201) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:50:37.384+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:50:37.386+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:50:37.386+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:50:37.441+0000] {logging_mixin.py:117} INFO - hola
[2022-11-07T15:50:37.455+0000] {processor.py:768} INFO - DAG(s) dict_keys(['forex_data_pipeline']) retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:50:37.502+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:50:37.501+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T15:50:37.525+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:50:37.525+0000] {dag.py:3328} INFO - Setting next_dagrun for forex_data_pipeline to 2022-11-07T00:00:00+00:00, run_after=2022-11-08T00:00:00+00:00
[2022-11-07T15:50:37.547+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.173 seconds
[2022-11-07T15:51:07.705+0000] {processor.py:156} INFO - Started process (PID=228) to work on /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:51:07.710+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/forex_data_pipeline.py for tasks to queue
[2022-11-07T15:51:07.711+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:51:07.711+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:51:07.770+0000] {logging_mixin.py:117} INFO - hola
[2022-11-07T15:51:07.782+0000] {processor.py:768} INFO - DAG(s) dict_keys(['forex_data_pipeline']) retrieved from /opt/airflow/dags/forex_data_pipeline.py
[2022-11-07T15:51:07.821+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:51:07.821+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T15:51:07.847+0000] {logging_mixin.py:117} INFO - [2022-11-07T15:51:07.846+0000] {dag.py:3328} INFO - Setting next_dagrun for forex_data_pipeline to 2022-11-07T00:00:00+00:00, run_after=2022-11-08T00:00:00+00:00
[2022-11-07T15:51:07.876+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/forex_data_pipeline.py took 0.174 seconds
